javascript/src/components/configurationforms/NumberField.jsx

```diff
@@ -16,15 +16,20 @@ var NumberField = React.createClass({
         this.setState(props);
     },
     mapValidationAttribute(attribute) {
-        switch(attribute) {
-            case "ONLY_NEGATIVE": return "negative_number";
-            case "ONLY_POSITIVE": return "positive_number";
-            case "IS_PORT_NUMBER": return "port_number";
-            default: return attribute.toLowerCase();
+        switch(attribute.toLocaleUpperCase()) {
+            case "ONLY_NEGATIVE": return {min: Number.MIN_SAFE_INTEGER, max: -1};
+            case "ONLY_POSITIVE": return {min: 0, max: Number.MAX_SAFE_INTEGER};
```

Comment:
We can't use `Number.MIN_SAFE_INTEGER` or `MAX_SAFE_INTEGER`, because IE doesn't support them.